testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveDatabricksUnityCompatibility.java

```diff
@@ -0,0 +1,94 @@
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.DATABRICKS_COMMUNICATION_FAILURE_MATCH;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.getLocationForSchema;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.getLocationForTable;
+import static io.trino.tests.product.utils.QueryExecutors.onDelta;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestHiveDatabricksUnityCompatibility
+        extends BaseTestDeltaLakeS3Storage
+{
+    private String unityCatalogCatalogName;
+    private String externalLocationPath;
+
+    @Override
+    @BeforeMethodWithContext
+    public void setUp()
+    {
+        super.setUp();
+        unityCatalogCatalogName = requireNonNull(System.getenv("DATABRICKS_UNITY_CATALOG_NAME"), "Environment variable not set: DATABRICKS_UNITY_CATALOG_NAME");
+        externalLocationPath = requireNonNull(System.getenv("DATABRICKS_UNITY_EXTERNAL_LOCATION"), "Environment variable not set: DATABRICKS_UNITY_EXTERNAL_LOCATION");
+    }
+
+    @Test(groups = {DATABRICKS_UNITY_HTTP_HMS, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testBasicHiveOperations()
+    {
+        String schemaName = "test_basic_hive_" + randomNameSuffix();
+        String tableName = "testTable";
+        try {
+            String schemaLocation = getLocationForSchema(bucketName, externalLocationPath, schemaName);
+            String tableLocation = getLocationForTable(bucketName, externalLocationPath, schemaName, tableName);
+            onDelta().executeQuery(format(
+                    "CREATE SCHEMA %s.%s MANAGED LOCATION '%s'",
+                    unityCatalogCatalogName, schemaName, schemaLocation));
+            onDelta().executeQuery(format(
+                    "CREATE TABLE %s.%s.%s (c1 int, c2 string) " +
+                            "USING PARQUET " +
+                            "LOCATION '%s'", unityCatalogCatalogName, schemaName, tableName, tableLocation));
+
+            onDelta().executeQuery(
+                    format(
+                            "INSERT INTO %s.%s.%s values (1, 'one')", unityCatalogCatalogName, schemaName, tableName));
+
+            assertThat(onTrino().executeQuery("SHOW SCHEMAS FROM hive"))
+                    .contains(ImmutableList.of(
+                            row(schemaName.toLowerCase(Locale.ROOT))));
+
+            assertThat(onTrino().executeQuery("SHOW TABLES in hive." + schemaName))
+                    .containsOnly(row("testtable"));
+
+            assertThat(onTrino().executeQuery(format("SELECT * FROM hive.%s.%s", schemaName, tableName)))
+                    .containsOnly(row(1, "one"));
+            /*
```

Comment:
Create an issue instead as a follow-up - no need to leave uncommented code here.